tests/test_unit_conversion.py

```diff
@@ -0,0 +1,19 @@
+import pytest
+import os
+import cellmlmanip
+
+OXMETA = "https://chaste.comlab.ox.ac.uk/cellml/ns/oxford-metadata"
```

Comment:
We should probably append a `#` here too so we're always being consistent about doing things properly ourselves!

Reply:
done